docs/dashboards/dashboards-overview.asciidoc

```diff
@@ -17,6 +17,8 @@ To create a new custom dashboard, click **Create Dashboard**. You can control wh
 
 To create a new tag or edit existing tags, open the **Tags** menu and click **Manage tags**.
 
+TIP: To make a custom dashboard appear on the security dashboards page, assign it the tag "Security solution".
```

Comment:
Bold might be more appropriate, since this appears as UI label text.
```suggestion
TIP: To make a custom dashboard appear on this page, assign it the tag **Security Solution**.
```